masterlist.yaml

```
@@ -26745,7 +26745,7 @@ plugins:
       - crc: 0x28033725
         util: 'SSEEdit v4.0.4c'
 
-  - name: 'Dawn of RedBag''s Solitude'
+  - name: 'Dawn of RedBag''s Solitude.esp'
```

I'll correct this in a separate commit as I'd rather keep this PR focused:
```suggestion
  - name: 'Dawn of RedBag''s Solitude'
```